addons/hr_holidays/static/src/persona_model_patch.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import { Persona } from "@mail/core/common/persona_model";
+import { patch } from "@web/core/utils/patch";
+
+patch(Persona.prototype, {
+    /**
+     * @overwrite
+     */
+    updateImStatus(newStatus) {
```

Comment:
`@overwrite` jsdoc is useless: we clearly see the super call in implementation. This just adds more lines in file, which can lead to more likely to scroll to see whole file content. The file is small but still no reason to make code more verbose than needed.